makefile

```diff
@@ -2,10 +2,10 @@ name := jasper
 buildDir := build
 srcFiles := $(shell find . -name "*.go" -not -path "./$(buildDir)/*" -not -name "*_test.go" -not -path "*\#*")
 testFiles := $(shell find . -name "*.go" -not -path "./$(buildDir)/*" -not -path "*\#*")
-packages := $(name) cli rpc rest options mock internal/executor
+packages := $(name) cli rpc rest options mock testutil internal-executor
```

Comment:
does this mean that we end up running tests on testutil?

Reply:
No, it only gets used for linting because `test-testutil` is not a task in evergreen.yaml.